### DISCUSSION TITLE
👽🤠🤖 Who is using Netmaker?

### DISCUSSION THREAD

**Discussion by afeiszli** [gravitl/netmaker#859] [Show and tell]
[👋](https://emojipedia.org/waving-hand/) Hey Netmaker Users,

If your company is using Netmaker in production, please comment below so we can mention you in the README / website.

Here's what your comment should look like:

Company: [add name here]
Permission to use logo in README/website: [Y/N]
Project description: [Short description of how you are using Netmaker]

Thank you!

**Comment by joy0630** [comment]
Please guide how to install it on smartphones
